test/e2e/set/remotekubernetescluster/config.go

```diff
@@ -1,9 +1,26 @@
 package remotekubernetescluster
 
-import "time"
+import (
+	"time"
+
+	"github.com/scylladb/scylla-operator/pkg/gather/collect"
+	"k8s.io/apimachinery/pkg/api/meta"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
 
 const (
 	testSetupTimeout    = 1 * time.Minute
 	testTeardownTimeout = 1 * time.Minute
-	testTimeout         = 3 * time.Minute
+	testTimeout         = 30 * time.Minute
+)
+
+var (
+	remoteKubernetesClusterResourceInfo = collect.ResourceInfo{
+		Resource: schema.GroupVersionResource{
+			Group:    "scylla.scylladb.com",
+			Version:  "v1alpha1",
+			Resource: "remotekuberneteclusters",
```

Comment:
typo: missing trailing `S` in `kubernete`.

I'm worried that this isn't failing the tests. Is this gvr necessary indeed?

Furthermore - the fact that resource and kind literals are scattered across the codebase feels like a code smell. Not something to fix in this PR, but worth noting.

Reply:
It's there to remove conflicting cluster-wide resource in case it's there and restore after the test. These resources are not created as part of CI so it just passes silently and it's expected. Good catch!

Used scyllav1alpha1.GroupVersion constants since we already have them, but left resource name since we don't have it anywhere.